core/tests/vm-benchmark/src/vm.rs

```diff
@@ -90,6 +93,29 @@ impl BenchmarkingVmFactory for Fast {
     ) -> Self::Instance {
         vm_fast::Vm::custom(batch_env, system_env, storage)
     }
+
+    fn count_instructions(tx: &Transaction) -> usize {
+        let mut vm = BenchmarkingVm::<Fast<InstructionCount>>::default();
```

Comment:
I had to create quite a type tangle here to make it work. Probably there's a better way that I can't think of right now.